sdks/java/core/src/main/java/org/apache/beam/sdk/util/FinishedTriggersBitSet.java

```diff
@@ -64,4 +64,9 @@ public void clearRecursively(ExecutableTrigger trigger) {
   public FinishedTriggersBitSet copy() {
     return new FinishedTriggersBitSet((BitSet) bitSet.clone());
   }
+
+  @Override
+  public String toString() {
+    return "FinishedTriggersBitSet" + bitSet.toString();
```

Comment:
Think this is going to come out weird. Instead of tweaking, perhaps use `MoreObjects.toStringHelper`.